lib/mayaUsd/ufe/UsdUndoVisibleCommand.cpp

```diff
@@ -43,7 +44,15 @@ void UsdUndoVisibleCommand::execute()
 {
     UsdUndoBlock undoBlock(&_undoableItem);
```

Comment:
Couple of nit-picks, do them if you feel like it.

I would move the UsdUndoBlock just before line 55, for clarity.

primImageAble is a weird CamelCase...

Reply:
Good points. Fait.

 9a3c05be3ca87b6a64b1f975cda68a3117fea01a